src/moveit_studio_agent_utils/package.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>moveit_studio_agent_utils</name>
+  <version>2.1.0</version>
+  <description>Package containing scripts for interacting with MoveIt Studio Agent</description>
+
+  <maintainer email="chance.cardona@picknik.ai">Chance Cardona</maintainer>
+  <author email="chance.cardona@picknik.ai">Chance Cardona</author>
+
+  <license>BSD-3</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <build_depend>moveit_common</build_depend>
```

Comment:
Is this necessary?

Reply:
It was not, good catch! It can be hard finding all of what's still leftover sometimes with these files I swear.